packages/demo/src/modules/clients/signals/addClient.js

```diff
@@ -6,6 +6,7 @@ export default [
   set(state`clients.$draft`, {}),
   set(state`clients.$draft.name`, state`clients.$filter`),
   set(state`clients.$draft.ref`, makeRef),
```

Comment:
Fell free to change. I do not know how to use these refs.
*I was using `const ref = firebase.database().ref('clients')` from firebase official lib to create these refs on my other projects.*